templates/ehub.json

```diff
@@ -66,6 +66,20 @@
                 "description": "Please use the actual name if using existing event hub."
             }
         },
+        "Event Hub Retention": {
+            "type": "Int",
+            "defaultValue": 7,
+            "metadata": {
+                "description": "Only applies if new event hub is created! Data retention time in days between 1 and 7."
```

Comment:
Perhaps use a similar format to earlier descriptions; something like:
"Optional. Data retention time in days, between 1 and 7. Leave empty if reusing an existing Event Hub."

Reply:
Actually, it's mandatory if creating a new event hub... And has a default value, so no "leave empty"...

So, maybe this:
"Mandatory if creating a new event hub. Data retention time in days, between 1 and 7."